{"uuid":"d1a3f835-d4b2-4798-b856-b638251413c1","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["4a80b7aa-be80-4ab4-b80e-2c159a93db79"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380651325,"stop":1673380651326}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380651332,"stop":1673380651332},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380651585,"stop":1673380651585},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380651737,"stop":1673380651737},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"3db30aae-c966-427a-ba93-3f7910fccebc-attachment","type":"image/png"},{"name":"Myscreenshot","source":"41983a67-3020-4725-82a3-f975b8b90287-attachment"}],"parameters":[],"start":1673380651752,"stop":1673380652078}],"start":1673380651325,"stop":1673380652079}